server/matchmaker/matchmaker_queue.py

```diff
@@ -288,9 +292,12 @@ def to_dict(self):
                 ndigits=2
             ),
             "num_players": self.num_players,
+            "ratings": sorted(ratings),
```

Comment:
Another thing that we've learned from experience over the years is that sending info in the form of a dict with descriptive key names is preferable over sending a list where you need to know what the data in the list means. This is because with dicts you can easily add more keys if you need to extend the message without breaking the existing functionality.

This might be fine in this case. I think we should rename `ratings` to something a bit more specific though as there is also a key in the `player_info` message called `ratings` which has a different structure from this. Maybe something like `rating_buckets`, `rating_groups` or something? Also would it be useful to send the granularity?

Reply:
maybe `active_rating_groups`? when I hear `rating_buckets` my first thought is that the list contains all eligible rating buckets and all buckets not listed are invalid, whatever that means.
I don't think it's useful to send the granularity. As long as the granularity is reasonably high it doesn't really matter for the client. And if we rounded to something unreasonably coarse like nearest 200 rating then the entire list would be useless anyway.